src/components/TextWidget.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react'
+import "../styles/text-widget.css"
+
+//Geckoboard- Article tag inline style is for updating position in drag and drop
+
+class TextWidget extends Component {
```

Comment:
Instead of naming this `TextWidget`, let's just call it `Text`.  It should be clear through usage that it is a widget.